Essentials/src/com/earth2me/essentials/RandomTeleport.java

```diff
@@ -175,6 +175,36 @@ public void cacheRandomLocations(Location center, double minRange, double maxRan
         return future;
     }
 
+    /**
+     * Returns an int array of length 2, representing a random coordinate pair that is both inside the square
+     * <pre>{@code x = maxRange * 2 and y = maxRange * 2 centered on (0,0)}</pre>
+     * and also NOT inside the square
+     * <pre>{@code x = minRange * 2 and y = minRange * 2 centered on (0,0)}</pre>
+     * The points generated by this method are evenly distributed throughout the area.
+     *
+     * @param minRange The minimum that {@code |x|} or {@code |y|} will be.
+     * @param maxRange The maximum that {@code |x|} or {@code |y|} will be.
+     * @return A pair of numbers, each randomly generated, that satisfy the above conditions.
+     */
```

Comment:
The rest of this file doesn't really have javadocs, and none of it is really API. If you can match the code style and turn this into a simple line (or multi-line) comment explaining what it does that would be cool. Otherwise we might want to think about adding javadocs to everything else here. 😛